### DISCUSSION TITLE
where to find `gowin_bba`?

### DISCUSSION THREAD

**Discussion by praveenivp** [YosysHQ/nextpnr#1265] [Q&A] [answered]
I'm trying to use sipeed tang primer 20k FPGA board. I'm using linux. I downloaded Gowin EDA (both IDE and programmer). I couldn't find any binary named `gowin_bba` within the folder. I'm completely new to FPGA. Can someone explain what is `gowin_bba` and where to find it?
Thanks!

**Comment by whitequark** [comment, marked as answer]
You need https://github.com/YosysHQ/apicula.

**Comment by praveenivp** [comment]
Thanks for the quick response. I was trying to install the dependencies first according to their documentation. I just noticed this line in their `setup.py` : `'gowin_bba=apycula.gowin_bba:main'`. I will install `apicula` before building `nextpnr`. Thanks.
